### ISSUE TITLE
GenericValue& operator[](const Ch* name) - bug if key not found

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/rapidjson#7]
```
This issue was closed by revision r83.
```

Original issue reported on code.google.com by `xushiweizh` on 6 Dec 2011 at 4:06

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 9b960b6b0af4.
```

Original comment by `milo...@gmail.com` on 6 Jun 2014 at 6:04
